README.md

```diff
@@ -11,6 +11,8 @@
 > of GitOps: the [GitOps Toolkit](https://toolkit.fluxcd.io/). In
 > consequence, we do not now plan to integrate GitOps Engine into
 > Flux.
+>
+> This also means that **[Flux v1 is in maintenance mode](https://github.com/fluxcd/flux/issues/3320)**.
```

Comment:
```suggestion
> :warning: This also means that **[Flux v1 is in maintenance mode](https://github.com/fluxcd/flux/issues/3320)**.
```

People have been having a hard time to find this notice, this may help?